### PR TITLE
Feature/core 702 onboarding feedback

### PR DESCRIPTION
**Pull request by Tschuck**
- onboarding
  - auto focus on company field
  - back buttons
  - Company placeholder nicht evan GmbH => Muster GmbH
  - input focus style
  - Sales Tax ID => VAT ID
  - Registration Number => Choose...
  - Recaptcha als letzte Checkbox
  - login password placeholder
  - terms of use design
  - company registration => switch registration page and contact page
  - remove registration page for non germany companies
- twin onboarding
  - images are hopping
  - rename gender to category
  - remove "vollständige" from history description
  - changes are allowed in future
  - i guess already known, but text on first page has around 3 times "Zwilling", 1 time "Twin", last page has 1 "Twin" and 1 "Zwilling"

### CODE REVIEW

File: dapps/evancore.vue.libs/src/components/forms/form/form.ts [hunk 1]
```diff
@@ -220,22 +220,32 @@ export default class EvanFormComponent extends mixins(EvanComponent) {
 
     // return directly specified translation
     let returnTranslation = type !== 'hint';
-    if (control.uiSpecs && control.uiSpecs.attr &&
-        control.uiSpecs.attr.hasOwnProperty(type)) {
+    if (control.uiSpecs &&
+        (control.uiSpecs.hasOwnProperty(type) ||
+          (control.uiSpecs.attr && control.uiSpecs.attr.hasOwnProperty(type)))
```
Comment (datrinh) [inline dapps/evancore.vue.libs/src/components/forms/form/form.ts]: I would consider refactoring into a small check function when you need 3 lines in the if-bracket

File: dapps/evancore.vue.libs/src/i18n/de.json [hunk 2]
```diff
@@ -89,7 +89,7 @@
     "not-your-account": "Nicht Ihr Account?",
     "optional": "optional",
     "password": "Passwort",
-    "password-placeholder": "Passwort von der ursprünglichen Profilerstellung",
+    "password-placeholder": "Passwort der Profilerstellung",
```
Comment (datrinh) [inline dapps/evancore.vue.libs/src/i18n/de.json]: Warum nicht einfach Passwort? Ist der Rest nicht redundant?

File: dapps/evancore.vue.libs/src/i18n/en.json [hunk 3]
```diff
@@ -89,7 +89,7 @@
     "not-your-account": "Not your account?",
     "optional": "optional",
     "password": "Password",
-    "password-placeholder": "Password from the original profile creation",
+    "password-placeholder": "Password from sign up",
```
Comment (datrinh) [inline dapps/evancore.vue.libs/src/i18n/en.json]: Same here

File: dapps/evancore.vue.libs/src/components/profile/profile-preview/profile-preview.ts [hunk 4]
```diff
@@ -219,4 +221,12 @@ export default class ProfilePreviewComponent extends mixins(EvanComponent) {
       FileHandler.fileToContainerFile(file)
     ));
   }
+
+  /**
+   * Overwrites the reset value, when the user cancels the edit mode.
+   */
+  setOriginalUserInfo() {
```
Comment (datrinh) [inline dapps/evancore.vue.libs/src/components/profile/profile-preview/profile-preview.ts]: I'd rename this. Functions starting with "set" will be expected to be setters. This is more of a "resetter" or "restorer".

**Review by datrinh — CHANGES_REQUESTED** [review]

File: dapps/evancore.vue.libs/src/components/forms/form/form.ts [hunk 5]
```diff
@@ -271,4 +268,24 @@ export default class EvanFormComponent extends mixins(EvanComponent) {
 
     return `evan-form-control-${ type }`;
   }
+
+  /**
+   * Overwrites a control a specific attribute.
```
Comment (datrinh) [inline dapps/evancore.vue.libs/src/components/forms/form/form.ts]: This comment wording seems confusing? :)

**Review by datrinh — CHANGES_REQUESTED** [review]

File: dapps/evancore.vue.libs/src/components/forms/form/form.ts (hunk 5 above)
Comment (Tschuck) [reply dapps/evancore.vue.libs/src/components/forms/form/form.ts]: Looks like i fall asleep while writing this :D

**Pull request merged**